chronos/models/usersChannels.js

```diff
@@ -1,13 +1,14 @@
 // @flow
-const { db } = require('./db');
+const { db } = require('shared/db');
 
 export const getUsersChannelsEligibleForWeeklyDigest = (
   userId: string
 ): Promise<Array<string>> => {
   return db
     .table('usersChannels')
-    .getAll(userId, { index: 'userId' })
-    .filter({ isMember: true })
+    .getAll([userId, 'member'], [userId, 'moderator'], [userId, 'owner'], {
```

Comment:
I've noticed you've done this check in a few places. If we're just trying to get members, why are you using all 3 indexes?

Reply:
Because owners and moderators are also members, but their roles are "owner" and "moderator"!

Reply:
This isn't true, so we can simplify this! If you are a moderator or owner you *also* are `isMember: true`

Reply:
We can't, unfortunately! 

Indexes can only have a single value, not multiple values. The "role" index is set to the highest one based on this precedence:

- Blocked
- Pending
- Owner
- Moderator
- Member

This means:

- If both `isMember` and `isOwner` are true, the users role will be `"owner"`.
- If both `isMember` and `isModerator` are true, the users role will be `"moderator"`.
- If both `isMember` and `isBlocked` are true (for whatever reason, bad db data most likely) the the role will be `"blocked"`.

That means if we want to get all members we also have to fetch the users who have moderator and owner roles, on top of the ones that have member roles.

Does that make sense?